gym/spaces/dict.py

```diff
@@ -101,8 +101,15 @@ def __init__(
         )  # None for shape and dtype, since it'll require special handling
 
     def seed(self, seed: Optional[Union[dict, int]] = None) -> list:
-        """Seed the PRNG of this space and all subspaces."""
+        """Seed the PRNG of this space and all subspaces.
+
+        This method will generate a (mostly) unique seed for each subspace. If a truly unique seed is required use .seed(list).
```

Comment:
Could you replace with below and for Tuple
```
Depending on the type of seed, the subspaces will be seeded differently
* None - All of the subspaces will use a random initial seed
* Int - The integer is used to seed the `Dict` space that is used to generate seed values for each of the subspaces. Warnings, this does not guarantee unique seeds for all of the subspaces. 
* Dict - Using all of the keys in the seed dictionary, the values are used to seed the subspaces. This allows the seeding of multiple composite subspaces (`Dict["space": Dict[...], ...]` with `{"space": {...}, ...}`).

Reply:
Updated!